pkg/gadgets/trace/capabilities/types/types.go

```diff
@@ -29,15 +32,29 @@ const (
 type Event struct {
 	eventtypes.Event
 
-	MountNsID uint64 `json:"mountnsid,omitempty"`
-	Pid       uint32 `json:"pid,omitempty"`
-	UID       uint32 `json:"uid,omitempty"`
-	Comm      string `json:"comm,omitempty"`
-	CapName   string `json:"capName,omitempty"`
-	Cap       int    `json:"cap,omitempty"`
-	Audit     int    `json:"audit,omitempty"`
-	InsetID   *bool  `json:"insetid,omitempty"`
-	Verdict   string `json:"verdict,omitempty"`
+	MountNsID uint64 `json:"mountnsid,omitempty" column:"mntns,width:12,hide"`
+	Pid       uint32 `json:"pid,omitempty" column:"pid,width:7,fixed"`
+	UID       uint32 `json:"uid,omitempty" column:"uid,width:10,fixed"`
+	Comm      string `json:"comm,omitempty" column:"comm,width:16,fixed"`
+	CapName   string `json:"capName,omitempty" column:"capName,width:20"`
```

Comment:
As [singal](https://github.com/kinvolk/inspektor-gadget/blob/172250945f743506f33f1878c03086cfd072ffc6/pkg/gadgets/trace/signal/types/types.go#L27) for trace/signal gadget, I suggest using `ellipsis:start` given that they start with `CAP_`:

```
$ ./kubectl-gadget trace capabilities -A
NODE          NAMESPACE     POD           CONTAINER    PID     UID        COMM             CAPNAME  CAP AUDIT VERDICT
master        gadget        gadget-78jxl  gadget       102891  0          gadgettracerman  CAP_BPF  39  1     Deny
master        gadget        gadget-78jxl  gadget       102891  0          gadgettracerman  CAP_SYS… 21  1     Allow
```

Reply:
Makes sense.